BepInEx.Core/BepInEx.Core.csproj

```diff
@@ -12,9 +12,6 @@
         <PackageReference Include="SemanticVersioning" Version="2.0.2"/>
         <PackageReference Include="MonoMod.Utils" Version="22.5.1.1"/>
     </ItemGroup>
-    <ItemGroup>
-        <Compile Remove="Contract\IPlugin.cs"/>
-    </ItemGroup>
```

Comment:
Should this be in this PR after all?

Reply:
With the compatibility accommodations, no so I reverted the changes